packages/edit-site/src/components/sidebar/style.scss

```diff
@@ -136,3 +136,10 @@ $split-border-control-offset: 55px;
 		}
 	}
 }
+
+// Override the `hr` styles defined for the complementary area component
```

Comment:
Short and sweet. Is this meant to be a temporary override to a point where the HR can be customized on the component level? If yes, then we might want to add a comment to that effect.

Reply:
Not really — the Global Styles Sidebar is already using `CardDivider` for this separator. The issue is that the sidebar internally uses a component from the `@wordpress/interface` package called `ComplementaryArea`, which introduces some style overrides, including [this one on all its children `hr`](https://github.com/WordPress/gutenberg/blob/c1ea8d60db9f05ebbfbaced50a88bfe30a7a0270/packages/interface/src/components/complementary-area/style.scss#L53-L57).

A few solutions that come to mind:

- Artificially increase the specificity of the `CardDivider` styles, so that they would not get overridden by the styles of `ComplementaryArea` — although I wouldn't recommend going down this route, it's a hacky approach that can cause a "race for specificity" in other parts of the repo;
- I guess the better solution would be to somehow rework how separators are used / styled in `ComplementaryArea`, although it may be a difficult task given how widespread the usage of `ComplementaryArea` may be.

Basically, the problem is that separators under `ComplementaryArea` are not encapsulated components, but are instead styles with some "global" css.